Resources/Prototypes/Floof/Entities/Objects/Weapons/guns.yml

```diff
@@ -76,3 +96,51 @@
   - type: Battery
     maxCharge: 1000
     startingCharge: 1000
+    
+- type: entity
+  name: energy shotgun
+  parent: BaseWeaponBattery
+  id: WeaponEnergyShotgun
+  description: A one-of-a-kind prototype energy weapon that uses various shotgun configurations. It offers the possibility of both lethal and non-lethal shots, making it a versatile weapon.
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/Battery/energy_shotgun.rsi
+    layers:
+      - state: base
+        map: ["enum.GunVisualLayers.Base"]
+      - state: mag-unshaded-4
+        map: ["enum.GunVisualLayers.MagUnshaded"]
+        shader: unshaded
+  - type: Clothing
+    sprite: Objects/Weapons/Guns/Battery/energy_shotgun.rsi
+  - type: Gun
+    fireRate: 2
+    soundGunshot:
+      path: /Audio/Weapons/Guns/Gunshots/laser_cannon.ogg
+  - type: ProjectileBatteryAmmoProvider
+    proto: eShellShotgun
+    fireCost: 150
+  - type: BatteryWeaponFireModes
+    fireModes:
+    - proto: eShellShotgun
+      fireCost: 150
+    - proto: BulletDisablerSmg
+      fireCost: 120
+  - type: Item
+    size: Large
+    shape:
+    - 0,0,3,1
+    sprite: Objects/Weapons/Guns/Battery/inhands_64x.rsi
+    heldPrefix: energy
+  - type: Tag
+    tags:
+    - HighRiskItem
+  - type: StealTarget
+    stealGroup: WeaponEnergyShotgun
+  - type: GunRequiresWield #remove when inaccuracy on spreads is fixed
+  - type: Battery
+    maxCharge: 1200
+    startingCharge: 1200
+  - type: BatterySelfRecharger
+    autoRecharge: true
+    autoRechargeRate: 24
```

Comment:
this is going to be hell for loof maintainers when they even try to merge the upcoming wizmerge from EE because you're creating a duplicate in a separate file.

Reply:
this is the same for the botany stuff floof has, good luck

Reply:
Sounds like more of a problem for downstream as we've started doing all our commits by hand.